patches/chrome-browser-ui-tabs-tab_strip_model.h.patch

```diff
@@ -0,0 +1,23 @@
+diff --git a/chrome/browser/ui/tabs/tab_strip_model.h b/chrome/browser/ui/tabs/tab_strip_model.h
+index f7282146e44d05a4f923adeed59089423bc6be62..1a92bed95ca16c45a982238c7cf5e78648125d40 100644
+--- a/chrome/browser/ui/tabs/tab_strip_model.h
++++ b/chrome/browser/ui/tabs/tab_strip_model.h
+@@ -384,8 +384,10 @@ class TabStripModel : public TabGroupController {
+   void CloseSelectedTabs();
+ 
+   // Select adjacent tabs
++  virtual
```

Comment:
This could be avoided by using chromium_src override, example: https://github.com/brave/brave-core/blob/9e7f5047afd7fdbb25aa884005918382f381578d/chromium_src/chrome/browser/ui/views/profiles/avatar_toolbar_button.h#L26